index.html

```diff
@@ -149,17 +149,7 @@ <h5>Adebayo Ngozi Ali</h5>
 
 		<!--PUT WHAT YOU COPIED BELOW HERE-->
 		
-		<div class="col-xs-6 col-sm-3 col-md-3">
-				<div class="wow bounceInUp" data-wow-delay="0.2s">
-                <div class="team boxed-grey">
-                    <div class="inner">
-						<div class="avatar"><img src="img/team/juliet.jpg" alt="" class="img-responsive img-circle" /></div>
-						<h5>Chigozie Ebere Juliet</h5>
-                        <p class="subtitle">Hi, I am Juliet, I am glad to be part of this course; and to be sincere, the journey has been worthwhile. I never knew git it this interesting at all; but this course has greatly enlightened me! I am Front-end devloper specialize in VUE.js as framework.</p>
-                    </div>
-                </div>
-				</div>
-			</div>
+		
```

Comment:
<div class="col-xs-6 col-sm-3 col-md-3">
				<div class="wow bounceInUp" data-wow-delay="0.2s">
                <div class="team boxed-grey">
                    <div class="inner">
						<div class="avatar"><img src="img/team/juliet.jpg" alt="" class="img-responsive img-circle" /></div>
						<h5>Chigozie Ebere Juliet</h5>
                        <p class="subtitle">Hi, my name is Bright David Adewale (Godson), I count it a great opportunity to be a part of this great program - D3 BOOTCAMP. The course has been very interesting and our instructor is a great man, because he's been very helpful and vast with impacting his knowledge. I'm aspiring to be Front-End Developer and good programmer with knowledge in HTML, CSS, and Javascript; as this will help me to impact my world.</p>
                    </div>
                </div>
				</div>
			</div>